docs/storage.md

```diff
@@ -471,6 +472,72 @@ config:
   directory: ""
 ```
 
+### OCI
+
+[Oracle Cloud Infrastructure](https://www.oracle.com/cloud). To configure OCI Object Storage as Thanos Object Store, you need access to OCI Object Storage.
+You can get a free OCI Cloud Services by following the [Oracle Cloud Free Tier](https://www.oracle.com/cloud/free) page.
```

Comment:
IMO, it's fine to link to docs like on line 477, 479, but this is more promotion / advertising. Let's keep this project neutral in that sense (i.e. we also don't add things about other cloud vendors free tiers).

Reply:
Hi @wiardvanrij 
Understand. I just pushed another commit with updated description. Please take a look. Thanks for reviewing it.

Reply:
Thanks, added a new review. I will also test it soon btw, just haven't got the time for it yet.

Reply:
@wiardvanrij we have an OCI tenancy available for CNCF project testing purposes, so if you can point me towards your test framework, we can start looking into providing the resources for CI-triggered automatic testing on an ongoing basis too.